build-support/emp3-gradle-plugin/src/main/resources/android-app.gradle

```diff
@@ -37,3 +38,10 @@ android {
         abortOnError false
     }
 }
+
+task testMe {
```

Comment:
Is this for debug purposes or should it remain in this file?

Reply:
Whoops, thats for debug purposes